{
  "candidate": "CVE-2019-6960",
  "description": "An issue was discovered in GitLab Community and Enterprise Edition 9.x, 10.x, and 11.x before 11.5.8, 11.6.x before 11.6.6, and 11.7.x before 11.7.1. It has Incorrect Access Control. Access to the internal wiki is permitted when an external wiki service is enabled.",
  "notes": {
    "msalvatore": "Affects GitLab 9.3 and later"
  },
  "priority": "untriaged",
  "cvss": {
    "nvd": " CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H [9.8 CRITICAL]"
  },
  "references": [
    "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-6960",
    "https://about.gitlab.com/2019/01/31/security-release-gitlab-11-dot-7-dot-3-released/"
  ],
  "public_date": "2019-09-09 20:15:00 UTC",
  "packages": {
    "bionic": {
      "gitlab": {
        "name": "gitlab",
        "status": "DNE"
      }
    },
    "cosmic": {
      "gitlab": {
        "name": "gitlab",
        "status": "DNE"
      }
    },
    "devel": {
      "gitlab": {
        "name": "gitlab",
        "status": "DNE"
      }
    },
    "disco": {
      "gitlab": {
        "name": "gitlab",
        "status": "DNE"
      }
    },
    "eoan": {
      "gitlab": {
        "name": "gitlab",
        "status": "DNE"
      }
    },
    "esm-apps/xenial": {
      "gitlab": {
        "name": "gitlab",
        "status": "not-affected",
        "note": "code not present"
      }
    },
    "precise/esm": {
      "gitlab": {
        "name": "gitlab",
        "status": "DNE"
      }
    },
    "trusty": {
      "gitlab": {
        "name": "gitlab",
        "status": "DNE"
      }
    },
    "trusty/esm": {
      "gitlab": {
        "name": "gitlab",
        "status": "DNE"
      }
    },
    "upstream": {
      "gitlab": {
        "name": "gitlab",
        "status": "needs-triage"
      }
    },
    "xenial": {
      "gitlab": {
        "name": "gitlab",
        "status": "not-affected",
        "note": "code not present"
      }
    }
  }
}
